client/src/app/auth/auth.service.ts

```diff
@@ -40,7 +40,7 @@ export class AuthService {
     });
   }
 
-  private setUser(userData: UserModel): void {
+  public setUser(userData: UserModel): void {
```

Comment:
why did you changed 'private' to 'public'?

Reply:
to check if it was called in test

Reply:
It means that your code depends on test? :)